otter/tap/api.py

```diff
@@ -286,6 +289,21 @@ def on_client_ready(_):
     return s
 
 
+def setup_converger(parent, kz_client, dispatcher):
+    """Create a Converger service."""
```

Comment:
@manishtomar: The Converger does create the partitioner internally (note that a factory is passed in, not a partitioner instance). If we passed in all the arguments instead, and later changed the implementation to not use a partitioner, we'd still need to change the __init__ signature to get rid of the useless arguments that are specific to ZK partitioning.